Header header
geographic_msgs/WayPoint[] WayPoints
geographic_msgs/RouteSegment[] RouteSegments
geographic_msgs/RoutePath[] RoutePaths 
